src/ansys/geometry/core/_grpc/_services/v0/conversions.py

```diff
@@ -49,6 +53,7 @@
     TrimmedSurface as GRPCTrimmedSurface,
 )
 from ansys.geometry.core.misc.checks import graphics_required
+from ansys.geometry.core.parameters.parameter import Parameter, ParameterType, ParameterUpdateStatus
```

Comment:
Move these imports inside type checking. And perform the import inside the methods as needed. This is critical